src/main/java/org/spongepowered/common/data/manipulator/immutable/entity/ImmutableSpongeSleepingData.java

```diff
@@ -0,0 +1,18 @@
+package org.spongepowered.common.data.manipulator.immutable.entity;
```

Comment:
Compile this with gradle as you have license violations. Will make sure tests complete too :).